Applications/SimpleShot/task_runner.cpp

```diff
@@ -3,10 +3,11 @@
  * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
  *
  * @file   task_runner.cpp
- * @date   08 Jan 2021
- * @brief  task runner for the simpleshot demonstration
+ * @date   23 Sep 2021
```

Comment:
Please do not change the date as this is not a new file.

Reply:
Okay

Reply:
Please Check now